### PR TITLE
Add scurrius

### PR DESCRIPTION
**Pull request by vkorn**
Adding new boss. Besides that added requirements file and removed unused imports

### CODE REVIEW

File: main_test.py [hunk 1]
```diff
@@ -99,6 +98,7 @@ def hiscores_raw_response():
 -1,-1
 -1,-1
 -1,-1
+1234,160
```
Comment (rmbarron) [inline main_test.py]: Nit: Please also update the documented total on l21

**Comment by rmbarron** [comment]
Thank you for the change! Only 1 minor documentation nit in a test before merging.

File: main_test.py (hunk 1 above)
Comment (vkorn) [reply main_test.py]: Updated

**Comment by vkorn** [comment]
Also decreased kc count from 80 to 60 as Nick requested in discord

**Review by rmbarron — APPROVED** [review]

**Pull request merged**